packages/vendure-plugin-stock-monitoring/src/ui/stock-widget.ts

```diff
@@ -58,4 +61,8 @@ export class StockWidgetComponent implements OnInit {
   imports: [SharedModule],
   declarations: [StockWidgetComponent],
 })
-export class StockWidgetModule {}
+export class StockWidgetModule {
+  reduceSum(stockLevels: any[]): number {
+    return stockLevels.reduce((acc, val) => (acc += val.stockOnHand), 0);
+  }
```

Comment:
Wow is it this simple to introduce global functions in Angular? Never knew, thats great :ok_hand: